### PR TITLE
Crpl 1719 expose express api using scripting action

### PR DESCRIPTION
**Pull request by danstocker**

### CODE REVIEW

File: boolean/reverse condition/4aca7b99-e216-476e-8e91-6d7d172fca16.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "name": "boolean",
+        "description": "Sends boolean depending on which input the signal was received through."
+      }
+    }
+  },
+  "name": "boolean/Reverse condition",
+  "attributes": {
+    "locked": true
+  },
+  "description": "Turns binary independent signals into a boolean.\n\nExample A:\n1. null@0 is received via `true`\n2. true@0 is sent via `boolean`\n\nExample B:\n1. \"foo\"@1 is received via `false`\n2. false@1 is sent via `boolean`",
```
Comment (lvass74) [inline boolean/reverse condition/4aca7b99-e216-476e-8e91-6d7d172fca16.json:35]: I think we already have a prototype for this purpose: `boolean/To boolean`

File: data/array/mapper/internal/7d61f7c0-a55f-4c1c-80ff-2b6fef8b70c4.json [hunk 2]
```diff
@@ -37,8 +37,9 @@
       }
     }
   },
-  "name": "data/array/Mapper/Internal",
+  "name": "data/array/Mapper/internal",
```
Comment (lvass74) [inline data/array/mapper/internal/7d61f7c0-a55f-4c1c-80ff-2b6fef8b70c4.json:40]: Is the lowercase intentional? All other internal nodes are called "Internal" with uppercase "I" at the moment.

**Review by lvass74 — COMMENTED** [review]
First comments after partial review. To be continued.

File: boolean/reverse condition/4aca7b99-e216-476e-8e91-6d7d172fca16.json (hunk 1 above)
Comment (danstocker) [reply boolean/reverse condition/4aca7b99-e216-476e-8e91-6d7d172fca16.json:35]: Forgot about it. I'll remove the new one, but adjust names / description on the existing one.

**Pull request merged**